src/foam/nanos/auth/PermissionTableView.js

```diff
@@ -73,7 +74,12 @@ foam.CLASS({
             .add('Groups: ').tag('br')
 
             .start().select(self.groupDAO.orderBy(self.Group.ID), function(g) {
-              groups[g.id] = foam.u2.md.CheckBox.create({label: g.id, data: true});
+              if ( self.user.group == g.id ) {
```

Comment:
This would be shorter as:
groups[g.id] = foam.u2.md.CheckBox.create({label: g.id, data: self.user.group == g.id});